### ISSUE TITLE
[Question] about the pre-trained wikidata embeddings

### ISSUE THREAD

**Issue by rahular** [facebookresearch/PyTorch-BigGraph#27]
Hi, thanks for open sourcing this useful tool! I wanted to know more details about the pre-trained wikidata embeddings. 

- What was the loss function used for training?
- What was the negative sampling scheme and how many negative samples were used?

Thanks!

**Comment by ledw** [comment]
@rahular 
* Loss function: softmax, metric: dot product. 
* negative sampling scheme: 500 uniform negatives and 500 batch negatives (for details of negative sampling scheme please refer to our [paper](https://arxiv.org/pdf/1903.12287.pdf)).
